brave-grpc/src/test/java/com/github/kristofa/brave/grpc/BraveGrpcInterceptorsTest.java

```diff
@@ -150,16 +150,22 @@ public void noSamplesWhenSamplingDisabled() throws Exception {
     void validateSpans() throws Exception {
         List<Span> spans = SpanCollectorForTesting.getInstance().getCollectedSpans();
         assertThat(spans.size(), is(equalTo(2)));
-        assertThat(spans.get(0).getTrace_id(), is(equalTo(spans.get(1).getTrace_id())));
-        validateSpan(spans.get(0), Arrays.asList("ss", "sr"));
+
+        Span serverSpan = spans.get(0);
+        assertThat(serverSpan.getTrace_id(), is(equalTo(spans.get(1).getTrace_id())));
+        validateSpan(serverSpan, Arrays.asList("ss", "sr"));
+
+        //Server spans should have the GRPC_REMOTE_ADDR binary annotation
+        Optional<BinaryAnnotation> binaryAnnotation =
```

Comment:
the value of this would be consistent, right? It might be nice to show what it is (ex ip:port)

here's some assertj slickness..

```java
    assertThat(span.binaryAnnotations)
        .filteredOn(b -> b.key.equals(Constants.SERVER_ADDR))
        .extracting(b -> new String(b.value))
        .containsOnly("host:port");
```